src/account_management.rs

```diff
@@ -0,0 +1,163 @@
+use crate::contacts::initial_sync;
+use crate::{
+    ACCOUNTS_DIR, Config, ensure_accounts_dir, get_account_store_path, list_accounts, open_store,
+};
+use anyhow::{Result, anyhow};
+use presage::Manager;
+use std::path::Path;
+
+pub async fn create_account_cli(account_name: String, device_name: String) -> Result<()> {
```

Comment:
```suggestion
pub async fn link_account_cli(account_name: String, device_name: String) -> Result<()> {
```

I don't think that "create" is an appropriate name here as we actually don't create the signal account here. My suggestion is "link".